src/numberAxis/Funnel.tsx

```diff
@@ -115,7 +116,9 @@ function getTooltipEntrySettings(props: FunnelProps): TooltipPayloadConfiguratio
 
 function FunnelTrapezoids(props: FunnelTrapezoidsProps) {
   const { trapezoids, shape, activeShape, allOtherFunnelProps } = props;
-  const { index: activeIndex } = useTooltipContext();
+  const activeItemIndex = useAppSelector(state =>
+    selectActiveIndex(state, 'item', state.tooltip.settings.trigger, undefined),
```

Comment:
hardcoded item cause I'm lazy and its the only thing funnel supports, can use the actual type later